sdk/v1/data_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	"testing"
 
 	sdk_args "github.com/newrelic/infra-integrations-sdk/args"
+	"github.com/stretchr/testify/assert"
```

Comment:
the same: https://github.com/newrelic/infra-integrations-sdk/pull/50#discussion_r181027556